app/services/send_welcome_email_service.rb

```diff
@@ -3,30 +3,30 @@
 class SendWelcomeEmailService
   include ServicePattern
 
-  def initialize(current_user:)
-    @current_user = current_user
+  def initialize(user:)
+    @user = user
```

Comment:
Changed the name of this variable because if this is being invoked by a sys admin then we really aren't sending this email to the `current_user`.